File: repos_estudiantes_TARQUI_tercera_entrega/ILLANES_ESCOBAR_FERNANDO_JOHSUA/PERSISTENCIA/Ejercicio1/Python/Main.py
from Empleado import Empleado
from GestorEmpleado import GestorEmpleado

gestor = GestorEmpleado()

e1 = Empleado("Luis", 30, 3000)
e2 = Empleado("Maria", 28, 4000)

gestor.guardar_empleado(e1)
gestor.guardar_empleado(e2)

print("Buscar por nombre: Maria")
print(gestor.busca_empleado("Maria"))

print("Empleado con salario mayor a 3500:")
print(gestor.mayor_salario(3500))

File: repos_estudiantes_TARQUI_tercera_entrega/ILLANES_ESCOBAR_FERNANDO_JOHSUA/PERSISTENCIA/Ejercicio1/Java/Main.java
package PERSISTENCIA.Ejercicio1.Java;

public class Main{
    public static void main(String[] args) {
        GestorEmpleado gestor = new GestorEmpleado();

        Empleado e1 = new Empleado("Luis", 30, 3000);
        Empleado e2 = new Empleado("Maria", 28, 4000);
        gestor.guardarEmpleado(e1);
        gestor.guardarEmpleado(e2);

        System.out.println("Buscar por nombre: Maria");
        System.out.println(gestor.buscaEmpleado("Maria"));

        System.out.println("Empleado con salario mayor a 3500:");
        System.out.println(gestor.mayorSalario(3500));
    }
}
